sklearn/manifold/_mds.py

```diff
@@ -324,6 +391,10 @@ class MDS(BaseEstimator):
             Pre-computed dissimilarities are passed directly to ``fit`` and
             ``fit_transform``.
 
+    method: string, optional, default: smacof
+        Methods for solving the MDS problem are "smacof" and "svd".
```

Comment:
The method used for solving the MDS problem.

Reply:
Done